### DISCUSSION TITLE
Help running Demo Application using Jaegar profile

### DISCUSSION THREAD

**Discussion by parabinda** [inspectIT/inspectit-ocelot#1219] [Q&A]
Hi,
As per instructions of running your demo Application for Ocelot agent, I could start them using docker-compose on my Ubuntu 16.04.

But after sometime the - Petclinic Application is not available at port 8080. 
It goes down. Please guide how to resolve this.
So, I cannot test the demo.

Also, please provide default userid and password for Various UIs.

1. Grafana Dashboard UI - what user/password to use
2. Config Server UI - what user/password to use

Regards
Arabinda

**Comment by mariusoe** [comment]
Hello @parabinda 

the passwords for Grafana and the Configuration Server are listed in the [documentation](https://inspectit.github.io/inspectit-ocelot/docs/getting-started/docker-examples#demo-scenarios). For all components, the username is `admin` and the password `demo`.

It's hard to say why the demo is not running on your machine. Ensure that enough memory is available because the demo starts a lot of containers. Especially when using Docker for Windows or Docker for MacOS, the default settings does not provide enough memory for the Docker VM.
Another possible problem could be that some container are waiting for each other. In case their dependencies are not started within 2min, they will failling. You can increase this timeout by modifying the `timeout=120s ` parameter in the docker compose file.
